.npmignore

```diff
@@ -2,4 +2,5 @@ tsconfig.json
 src
 test
 .idea
-.vscode
\ No newline at end of file
+.vscode
+.github
```

Comment:
If we use GitHub actions, we want to keep this folder. I know GitHub actions are not working now so maybe you can simply delete the folder instead of putting it in Gitignore?